homeassistant/components/knx/storage/entity_store_schema.py

```diff
@@ -57,6 +62,8 @@
         vol.Optional(CONF_ENTITY_CATEGORY, default=None): vol.Any(
             ENTITY_CATEGORIES_SCHEMA, vol.SetTo(None)
         ),
+        vol.Optional(CONF_STATE_CLASS, default=None): vol.Maybe(str),
+        vol.Optional(CONF_DEVICE_CLASS, default=None): vol.Maybe(str),
```

Comment:
`state_class` and `device_class` are sensor platform specific and should be validated by their respective schema instead of `str`.